tests/parm/input_global_parent.nml.IN

```diff
@@ -0,0 +1,410 @@
+  dz_min =  @[DZ_MIN]
+  grid_type = 0
+  make_nh = @[MAKE_NH]
+  fv_debug = .false.
+  range_warn = .true.
+  reset_eta = .false.
+  n_sponge = @[N_SPONGE]
+  n_zs_filter = @[N_ZS_FILTER]
+  nudge_qv = .false.
+  nudge_dz = .false.
+  tau = @[FV_CORE_TAU]
+  rf_cutoff = @[RF_CUTOFF]
+  rf_fast = @[RF_FAST]
+  sg_cutoff = @[SG_CUTOFF]
+  upoff = @[UPOFF]
+  d2_bg_k1 = @[D2_BG_K1]
+  d2_bg_k2 = @[D2_BG_K2]
+  kord_tm = -9
+  kord_mt = 9
+  kord_wz = 9
+  kord_tr = 9
+  hydrostatic = .false.
+  phys_hydrostatic = .false.
+  use_hydro_pressure = .false.
+  beta = 0.
+  a_imp = 1.
+  p_fac = 0.1
+  k_split = 1
+  n_split = 5
+  nwat = @[NWAT]
+  na_init = @[NA_INIT]
+  d_ext = @[D_EXT]
+  dnats = @[DNATS]
+  fv_sg_adj = @[FV_SG_ADJ]
+  d2_bg = @[D2_BG]
+  nord = 2
+  nord_zs_filter = @[NORD_ZS_FILTER]
+  dddmp = @[DDDMP]
+  d4_bg = @[D4_BG]
+  vtdm4 = @[VTDM4]
+  delt_max = @[DELT_MAX]
+  ke_bg = 0.
+  do_vort_damp = .true.
+  external_ic = @[EXTERNAL_IC]
+  external_eta = .true.
+  gfs_phil = .false.
+  nggps_ic = @[NGGPS_IC]
+  mountain = @[MOUNTAIN]
+  ncep_ic = .false.
+  d_con = @[D_CON]
+  hord_mt = @[HORD_MT]
+  hord_vt = @[HORD_VT]
+  hord_tm = @[HORD_TM]
+  hord_dp = @[HORD_DP]
+  hord_tr = @[HORD_TR]
+  adjust_dry_mass = .false.
+  dry_mass = @[DRY_MASS]
+  consv_te = @[CONSV_TE]
+  do_sat_adj = @[DO_SAT_ADJ]
+  fast_tau_w_sec = @[FAST_TAU_W_SEC]
+  consv_am = .false.
+  fill = .true.
+  full_zs_filter = @[FULL_ZS_FILTER]
+  dwind_2d = .false.
+  print_freq = 6
+  warm_start = @[WARM_START]
+  no_dycore = .false.
+  z_tracer = .true.
+  agrid_vel_rst = .true.
+  read_increment = @[READ_INCREMENT]
+  res_latlon_dynamics = @[RES_LATLON_DYNAMICS]
+  do_schmidt = .true.
+  target_lat = @[TARGET_LAT]
+  target_lon = @[TARGET_LON]
+  stretch_fac = 1.0001
+/
+
+&external_ic_nml
+  filtered_terrain = .true.
+  levp = 128
+  gfs_dwinds = .true.
+  checker_tr = .false.
+  nt_checker = 0
+  
+/
+
+&surf_map_nml
+  cd2 = -1
+  cd4 = 0.15
+  max_slope = 0.15
+  n_del2_strong = 0
+  n_del2_weak = 15
+  n_del4 = 2
+  peak_fac = 1.
+  zero_ocean = .false.
+/
+
+&gfs_physics_nml
+  fhzero       = @[FHZERO]
+  h2o_phys     = @[H2O_PHYS]
+  ldiag3d      = @[LDIAG3D]
+  qdiag3d      = @[QDIAG3D]
+  lightning_threat = @[LIGHTNING_THREAT]
+  print_diff_pgr = @[PRINT_DIFF_PGR]
+  fhcyc        = @[FHCYC]
+  use_ufo      = .true.
+  pre_rad      = .false.
+  imp_physics  = @[IMP_PHYSICS]
+  iovr         = @[IOVR]
+  lcnorm       = @[LCNORM]
+  ltaerosol    = .true.
+  lradar       = .true.
+  ttendlim     = -999
+  dt_inner     = @[DT_INNER]
+  sedi_semi    = @[SEDI_SEMI]
+  decfl        = @[DECFL]
+  oz_phys      = @[OZ_PHYS_NEW]
+  oz_phys_2015 = @[OZ_PHYS_OLD]
+  lsoil_lsm    = @[LSOIL_LSM]
+  do_mynnedmf  = @[DO_MYNNEDMF]
+  do_mynnsfclay = @[DO_MYNNSFCLAY]
+  icloud_bl    = 1
+  bl_mynn_edmf = 1
+  bl_mynn_tkeadvect = @[BL_MYNN_TKEADVECT]
+  bl_mynn_edmf_mom = 1
+  do_ugwp      = @[DO_UGWP]
+  do_tofd      = @[DO_TOFD]
+  do_ca        = @[DO_CA]
```

Comment:
The current global_workflow does not have line 160

Reply:
It has to; that's the flag to turn on or off cellular automata. Please recheck.

Reply:
Sorry, I should made it clear that what I mean is that the current global workflow does not "explicitly" set do_ca. It takes whatever default value.

Reply:
I thought the global-workflow scripts have a DO_CA flag to turn off cellular automata. Are you telling me that flag is not connected to the namelist? That sounds like a bug to me...

Reply:
Sorry for the confusion. Yes, you are right. The current global workflow does have DO_CA=.true.

The current global workflow hasn't used an input.nml yet. It uses parsing_namelists_FV3.sh to generate a namelist. For nesting runs, DO_CA is set to NO and hence all DO_CA related namelist options are missing:
```
if [[ ${DO_CA:-"NO"} = "YES" ]]; then
  cat >> "${nml_file}" << EOF
  do_ca      = .true.
  ca_global  = ${ca_global:-".false."}
  ca_sgs     = ${ca_sgs:-".true."}
  nca        = ${nca:-"1"}
  ncells     = ${ncells:-"5"}
  nlives     = ${nlives:-"12"}
  nseed      = ${nseed:-"1"}
  nfracseed  = ${nfracseed:-"0.5"}
  nthresh    = ${nthresh:-"18"}
  ca_trigger = ${ca_trigger:-".true."}
  nspinup    = ${nspinup:-"1"}
  iseed_ca   = ${ISEED_CA:-"12345"}
EOF
```

Reply:
Do we need to set DO_CA to true for the parent domain and false for the nest domain? Or false for both domains?

Reply:
The current cellular automata implementation does not support nesting. It must be off for all tiles when nests are in use.

Reply:
ok, good.

Reply:
@guoqing-noaa you should turn do_ca = .false. for the nest. It is default true for GFS and GEFS.

Reply:
Due to the upscale feedback, if the nest fields have invalid data (like all zero) then they'll replace the parent arrays. If you want to turn off CA for the nest, it has to be off for the globe too.